src/go/pt-galera-log-explainer/regex/file_test.go

```diff
@@ -0,0 +1,47 @@
+package regex
+
+import (
+	"testing"
+)
+
+func TestFileType(t *testing.T) {
```

Comment:
```suggestion
func TestFileType(t *testing.T) {
    t.Parallel()
```

Reply:
Done (in another commit, among others things)